sql-indent-test.el

```diff
@@ -342,4 +342,7 @@ information read from DATA-FILE (as generated by
 (ert-deftest sqlind-ert-pr46 ()
   (sqlind-ert-check-file-syntax "test-data/pr46.sql" "test-data/pr46-syn.eld"))
 
+(ert-deftest sqlind-ert-pr46 ()
```

Comment:
Can you fix the name of the test?  It should be `sqlind-ert-pr48`.  Your definition erases the previous one.

Reply:
Ooops, sorry.